src/app/Login/page.module.css

```diff
@@ -22,35 +26,108 @@
 }
 
 .welcomeBackcontainer {
-  margin: 100px;
-  text-align: center;
-  font-size: 50px;
-  text-align: center;
-  color: #ffffff;
-  margin-bottom: 100px;
-}
-
-.loginPageContainer {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  height: 100vh;
-  width: 100vw;
+  margin: 3rem;
+  font-size: 3rem;
+  margin-bottom: 3rem;
 }
-
 .lgnbttncontainer,
 .signbttncontainer {
   display: flex;
   justify-content: center;
   align-items: center;
-  margin-top: 20px;
+  margin-top: 1rem;
 }
 
 .Err {
   color: #e74c3c;
   font-size: 14px;
-  margin-top: 5px;
+  margin-top: 1rem;
   font-weight: bold;
   display: flex;
   align-items: center;
 }
+@keyframes shake {
+  0% {
+    transform: translateX(0);
+  }
+  25% {
+    transform: translateX(-5px);
+  }
+  50% {
+    transform: translateX(5px);
+  }
+  75% {
+    transform: translateX(-5px);
+  }
+  100% {
+    transform: translateX(0);
+  }
+}
+
+.Err.shake {
+  animation: shake 0.2s ease; /* Adjust duration and timing as needed */
+}
```

Comment:
Since these classes are all specifically for the text field and no other component, it would be better to give this CSS to only the text field component so this does not have to get repeated if textfield is also added on another page like settings